meteoroid/core/libs/clients/open_whisk_client.py

```diff
@@ -1,13 +1,14 @@
 import json
 import os
 import requests
+import logging
```

Comment:
The ideal order of the import syntax is as follows:
1. Python standard library
2. Third party library
3. Custom library

https://www.python.org/dev/peps/pep-0008/#id23

Can it be fixed as follows?
```
import json
import logging
import os

import requests
```